plugins/modules/nd_service_instance.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Sabari Jaganathan (@sajagana) <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_service_instance
+short_description: Manages Service Instance on Nexus Dashboard.
+description:
+- Manages Service Instance of the Nexus Dashboard.
+author:
+- Sabari Jaganathan (@sajagana)
+options:
+  name:
+    description:
+    - The name of the Service.
+    aliases: [ service_name ]
+    type: str
+  instance_name:
+    description:
+    - The name of the Service Instance Profile.
```

Comment:
```suggestion
    - The name of the Service Instance.
```

Reply:
should this be just service instance, because I see no references to profile anywhere int the rest of the code or docs